runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -1188,6 +1188,270 @@ TR::Register *J9::Power::TreeEvaluator::awrtbariEvaluator(TR::Node *node, TR::Co
    return NULL;
    }
 
+TR::Register *J9::Power::TreeEvaluator::irdbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT(TR::Compiler->om.shouldGenerateReadBarriersForFieldLoads(), "iReadBarrierEvaluator");
```

Comment:
What about field watch? This assert doesn't seem correct.

Reply:
We have not implemented Field Watch yet. We are working on it.